app/views/spree/checkout/payment/_affirm.html.erb

```diff
@@ -173,6 +167,15 @@
           }
         });
       });
+
+      /*****************************************************\
+          remove the payment source empty box from UI
+      \*****************************************************/
+      $(document).on('click', '#checkout_form_payment'
+        , function (event) {
+          if($('div[data-hook="checkout_payment_step"] input[type="radio"]:checked')[0].parentNode.outerText === "Affirm Monthly Payments") $('.payment-sources').hide();
```

Comment:
This looks like it hides everything with a class name of "payment-sources" is that correct?

Reply:
That is correct. Payment sources is a div that is not needed for the Affirm payment method